### ISSUE TITLE
Document and learn from the FNS NAC "Walking Skeleton" Approach to ATO

### ISSUE THREAD

**Issue by alexbielen** [18F/TLC-crew#178]
### A description of the work
18F's engagement with the USDA FNS NAC project recently ended. This was a multi-year engagement that involved new approaches to many government delivery issues. In support of 18F's Objective 2 ("know, share, and evolve our best practices") we'd like to reflect on several of the approaches that the USDA FNS NAC team used over the course of the project. 

The first approach that we'd like to reflect on is the "Walking Skeleton" approach to the ATO. 

From the team's Midpoint Reflection (1/23/23)

> Frontloading the initial ATO resulted in rework. In order to mitigate the schedule risk of an ATO to meet a Congressional deadline, the team tried to speed up the security review process by creating a minimal viable product called a “walking skeleton.” This strategy accomplished scoping FNS’s security requirements, but we’re blocked by non-compliance barriers to launch (i.e. rule publication, SORN). Once security vulnerabilities were identified, remediating them became a priority and for subsequent iterations of the product, the security team required repeat assessments.

In support of this, we'd like to plan a one hour reflection with USDA FNS NAC team members specifically on the "Walking Skeleton" approach. We will have a third-party facilitator and note-taker available so that the team can focus on reflecting. 

After the reflection, we'd like to create a shareable artifact (e.g., presentation, blog post, project resource etc) to capture lessons learned from the "Walking Skeleton" approach. 

### Point of contact on this issue
Who can we follow-up with if we have questions?
@alexbielen 

**Billable?**

- [ ] Yes
- [x] No
If yes, tock code:

**Skills needed**

- 18F's FNS NAC Team Members (@kingcomma @cjhskippy @edwintorres @ecayer and others as needed)
- Facilitator @jskinne3 
- Note-taker (any chapter)
- Content Strategist

**Timeline**
Does this need to happen in the next two weeks?

- [x] Yes
- [ ] No

How much time do you anticipate this work taking?
5-10 hours? I'm not sure. A few hours for planning the reflection, 1-1.5 hour for reflection, 5-8 hours for producing an artifact. 

### Acceptance Criteria

- [x] Reflection is scheduled and planned
- [ ] Reflection is completed and documented
- [ ] Lessons learned are shared in a durable artifact 

We could break out the "Lessons learned are shared in a durable artifact" into a separate ticket as appropriate.

**Comment by kingcomma** [comment]
Hoping this reflection can capture:

- Why did the team choose this approach?
- Did the approach work for that purpose?
- Tradeoffs / potential drawbacks of the approach
- Could the approach work for other 18F teams on future projects?
  - Under what circumstances, if any?

**Comment by kingcomma** [comment]
Initial reflection scheduled for 5/12. Expecting to find a facilitator during the next increment planning meeting on 5/9.

**Comment by jskinne3** [comment]
Initial reflection [document](https://docs.google.com/document/d/1HxPBLgUTAVvBEq_81xtqNfrYhyeTIaXOvm16SZv0Cy0/) created today
